src/components/Choropleth.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+    <div v-if="initialising">loading..</div>
+    <div v-else>
+        <LMap ref="map" style="height: 100vh; width: 100%" @ready="updateBounds">
+            <LTileLayer
+                :url="backgroundLayer.url"
+                :attribution="backgroundLayer.attribution"
+                :max-zoom="backgroundLayer.maxZoom"
+                :min-zoom="backgroundLayer.minZoom"
+            ></LTileLayer>
```

Comment:
i think
```suggestion
            <LTileLayer v-bind="backgroundLayer"></LTileLayer>
```

Reply:
Great thanks, I forgot you could do this! :)